assets/prototype/application/services/context/ConfigProvider.tsx

```diff
@@ -26,4 +26,4 @@ const ConfigProvider: React.FunctionComponent<ProviderProps> = ({ children }): J
 	return <Provider value={{ config, setConfig }}>{children}</Provider>;
 };
 
-export default ConfigProvider;
```

Comment:
Originally it was exporting `ConfigProvider` as default because it would be a little bit confusing to destructure `ConfigProvider` from `ConfigProvider`
import { ConfigProvider } from '../ConfigProvider';

Reply:
We need index files for many such cases, may be another PR?